doc/conf.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
You should have a look at https://github.com/dftlibs/xcfun/blob/master/docs/conf.py I simplified it a lot with respect to this version here.